components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/store/v1/impl/SubscriptionsApiServiceImpl.java

```diff
@@ -37,6 +37,7 @@
 import org.wso2.carbon.apimgt.api.model.SubscribedAPI;
 import org.wso2.carbon.apimgt.api.model.Subscriber;
 import org.wso2.carbon.apimgt.api.model.SubscriptionResponse;
+import org.wso2.carbon.apimgt.impl.*;
```

Comment:
Why do we have this wildcard import here? Shall we remove it?

Reply:
Fixed in - https://github.com/wso2/carbon-apimgt/pull/11699